### PR TITLE
Release v0.27.1

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/blog/releases/v0.27.1.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## Changelog since v0.27.0
+
+### Notable Changes
+
+* Add labels information to deployment detail page ([#3423](https://github.com/pipe-cd/pipecd/pull/3423))
+
+### Internal Changes
+
+* Fix the examples ([#3436](https://github.com/pipe-cd/pipecd/pull/3436))
+* Fix help buttom position ([#3434](https://github.com/pipe-cd/pipecd/pull/3434))
```
Comment (knanao) [inline docs/content/en/blog/releases/v0.27.1.md:18]: Sorry, Please fix typo `buttom` into `button`.

Comment (nghialv) [reply docs/content/en/blog/releases/v0.27.1.md]: Nice catch! Fixed.

**Comment by knanao** [comment]
Thank you!
/lgtm

**Comment by khanhtc1202** [comment]
🙌
/approve

**Pull request merged**